README.md

```diff
@@ -1 +1,163 @@
-# math_react_app
\ No newline at end of file
+<a name="readme-top"></a>
+<div align="center">
+
+  <br/>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+  - [Kanban Board](#kanban)
+  - [Explaining Project Video](#project-video)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [⭐️ Show your support](#support)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Kanban Restaurant Project] <a name="about-project"></a>
+
+> This project is created for restaurant that people can only give commend and give like to meals that the want. In this project we have used the mealDB API [https://www.themealdb.com/api.php] for displying the items in the page, for like and comment part of the project we have used the Involvement API [https://www.notion.so/Involvement-API-869e60b5ad104603aa6db59e08150270]. We have added 7 test cases for this project up to now.
+
+## 🛠 Built With <a name="built-with"></a>
+  <ul>
+    <li><a href="https://www.w3schools.com/html">HTML</a></li>
+    <li><a href="https://www.w3schools.com/CSS">CSS</a></li>
+    <li><a href="https://www.w3schools.com/js/">JS</a></li>
+    <li><a href="https://webpack.js.org/">webPack</a></li>
+    <li><a href="https://jestjs.io/">jest for testing</a></li>
+    <li><a href="https://www.themealdb.com/api.php">mealDB API</a></li>
+    <li><a href="https://www.notion.so/Involvement-API-869e60b5ad104603aa6db59e08150270">Involvement API</a></li>
+  </ul>
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[Display All Items from mealDB API]**
+- **[Add New Comment]**
+- **[Like meals]**
+- **[Display number of like for diffrent Meals]**
+- **[Display number of comments for diffrent Meals]**
+- **[Fully Created with ES6]**
+- **[Fullty tested with jest testing framework]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Kanban Borad <a name="kanban">https://github.com/Hidar37/kanban_restaurant_project/projects/1</a>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Explaining Project Video <a name="project-video">https://drive.google.com/file/d/1RYCKD4TH2wGO8_fK4P4JL2dnyfrqZ-8F/view?usp=sharing</a>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- [ ] I think you used an old README file from a past project without editing it with the new project data, Please edit every section of your readme file to fit the new project.

Reply:
I did not update me **readme** file because the **requirement** was not to **updated** the **readme** file. The **propriety** was to **setup** the project with **React**. That is why I did not **update** me **readme** file.